Project21/Project21/conversionfile.c

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment 3 ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.Naveenkumar s
+// ------------------------------------------------------------------
+// Assignment 3
+// Decimal to Binary conversion and Decimal to HexaDecimal conversion
+// ------------------------------------------------------------------------------------------------
+#define INT_SIZE sizeof(int)*8
+#define HEX_SIZE 8
+
+char binary[INT_SIZE + 1];

```

Comment:
Please avoid global variables.

Reply:
Please avoid marking conversations as resolved unless you have made the changes @Naveen-2611 
Also, Global variables are avoided to prevent bugs caused by unrestricted access to the variable allowing everyone to edit the values in it. It makes it difficult to keep track of all possible changes and makes the code more complex. See [here](https://stackoverflow.com/questions/176118/when-is-it-ok-to-use-a-global-variable-in-c).